indico/modules/events/editing/models/tags.py

```diff
@@ -66,6 +66,10 @@ def verbose_title(self):
         """Properly formatted title, including tag code."""
         return f'{self.code}: {self.title}'
 
+    def log(self, *args, **kwargs):
+        """Log with prefilled metadata for the tag."""
+        return self.event.log(*args, meta={'tag_id': self.id}, **kwargs)
```

Comment:
```suggestion
        return self.event.log(*args, meta={'editing_tag_id': self.id}, **kwargs)
```

We also have registration tags and if we were to log anything for those, they should not "collide" (show up both) when filtering the event logs by metadata.